runtime/lib/backend/common/Utils.hpp

```diff
@@ -198,6 +200,8 @@ constexpr std::array simulator_gate_info = {
     GateInfoTupleT{SimulatorGate::CY, "CY", 2, 0},
     GateInfoTupleT{SimulatorGate::CZ, "CZ", 2, 0},
     GateInfoTupleT{SimulatorGate::SWAP, "SWAP", 2, 0},
+    GateInfoTupleT{SimulatorGate::ISWAP, "ISWAP", 2, 0},
+    GateInfoTupleT{SimulatorGate::PSWAP, "PSWAP", 2, 1},
```

Comment:
What are your thoughts about removing these utility arrays? What would you do if you wanted to re-implement the gate support in the runtime from the beginning?